web/js/components/layer/product-picker/browse/browse-layers.js

```diff
@@ -167,6 +171,11 @@ function BrowseLayers (props) {
       );
   }
 
+  function renderSelectedCategoryName() {
```

Comment:
This is fine but since there's not a lot of complexity in this render function, it could just be handled in one line in the `renderMobileDropdown()` function:

```
{selectedCategoryName && <div className="selected-category">{selectedCategoryName}</div>}
```

Reply:
Resolved in https://github.com/nasa-gibs/worldview/pull/3783/commits/50b1ef41dfb4c212ed2a1b37313a18b937f25ad6